### PR TITLE
Widget fixes

### PR DESCRIPTION
**Pull request by ghost**
As reported in #52 
- Fix unicode error in select shortcut dialog when generating widgetName
- Fix skin-localised strings displaying in management dialog

### CODE REVIEW

File: resources/lib/library.py [hunk 1]
```diff
@@ -1500,9 +1501,9 @@ def explorer( self, history, location, label, thumbnail, itemType, isWidget = Fa
         widgetType = None
         addonType = None
 
-        dialogLabel = label[0].replace( "  >", "" )
+        dialogLabel = try_decode( label[0] ).replace( "  >", "" )
         if len( label ) != 1:
-            dialogLabel = label[0].replace( "  >", "" ) + " - " + label[ len( label ) - 1 ].replace( "  >", "" )
+            dialogLabel = try_decode( label[0] ).replace( "  >", "" ) + " - " + try_decode( label[ len( label ) - 1 ] ).replace( "  >", "" )
```
Comment (phil65) [inline resources/lib/library.py:1507]: pythonic way for "label[ len( label ) - 1 ]" is "label[:-1]" (you want to remove the last char, right?)

Comment (ghost) [inline resources/lib/library.py:1507]: `label` is actually an array of strings, so it's not removing the last char, it's getting the last element of the array. Probably still not doing it the pythonic way, though ;)

Comment (phil65) [inline resources/lib/library.py:1507]: in that case "label[-1]" ;)

Comment (ghost) [inline resources/lib/library.py:1507]: Y'know, I'm starting to hate Python every time I return to it ;) Cheers Phil I'll update it right away.

**Comment by marcelveldt** [comment]
Thanks for fixing!

**Pull request merged**